data/mods/Xedra_Evolved/player/professions.json

```diff
@@ -263,6 +262,190 @@
+          { "item": "fn57_sup", "ammo-item": "57mm_ss", "charges": 20, "container-item": "shoulder_holster" },
+          { "item": "legpouch_large", "contents-group": "suppressed_fn57" }
+        ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "sports_bra", "boxer_shorts" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "xe_eater",
+    "name": "Mobile Taks Force: Upsilon 4 \"Devourers\"",
+    "description": "Being part of Upsilon 4, you were always at the front of your unit, protecting other operatives from danger ahead using your eater's abilities.  This one was no different, but something went really wrong.",
+    "points": 7,
+    "skills": [
+      { "level": 2, "name": "gun" },
+      { "level": 2, "name": "shotgun" },
+      { "level": 1, "name": "dodge" },
+      { "level": 2, "name": "melee" },
+      { "level": 3, "name": "deduction" }
+    ],
+    "spells": [
+      { "id": "spring_heeled_leap", "level": 7 },
+      { "id": "spell_dodge", "level": 7 },
+      { "id": "spell_endurance", "level": 7 }
+    ],
+    "traits": [ "EATER" ],
+    "items": {
+      "both": {
+        "ammo": 100,
+        "items": [
+          "pants_army",
+          "undershirt",
+          "combat_shirt",
+          "tac_fullhelmet",
+          "armguard_hard",
+          "legguard_hard",
+          "mask_ski",
+          "gloves_liner",
+          "gloves_tactical",
+          "socks",
+          "boots_combat",
+          "wristwatch",
+          "molle_pack",
+          "hammer_sledge"
+        ],
+        "entries": [
+          { "item": "xedra_jumpsuit", "variant": "xedra_jumpsuit_eater" },
+          { "group": "charged_two_way_radio" },
+          { "group": "us_ballistic_vest_pristine" },
+          { "item": "water_clean", "container-item": "canteen" },
+          { "item": "mossberg_500", "ammo-item": "shot_00", "charges": 6, "contents-item": [ "shoulder_strap" ] },
+          { "item": "ear_plugs", "custom-flags": [ "no_auto_equip" ] },
+          { "item": "grenadebandolier", "contents-item": [ "flashbang", "flashbang" ] },
+          { "item": "bandolier_shotgun", "contents-group": "bandolier_swat_cqc1" },
+          { "item": "knife_combat", "container-item": "sheath" },
+          { "item": "m17", "ammo-item": "9mm", "charges": 17, "container-item": "holster" }
+        ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "sports_bra", "boxer_shorts" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "xe_dreamer",
+    "name": "Mobile Taks Force: Theta 1 \"Middle-Management\"",
+    "description": "Theta 1, also known as \"kindergarten\" between other task forces, is used when missions are expected to be too dangerous to send people or when reconnaissance can't obtain any important information about the target, which is why otherworldly creatures, summoned by dreamers, are sent.  It's not the best course of action, but the unit's nearly zero fatality rate makes it worth using.",
```

Comment:
>It's not the best course of action, but the unit's nearly zero fatality rate makes it worth using.

So why is it not the best course of action, collateral damage? 🤔

Reply:
Their intellectual abilities do not match their strength, that's why the only thing summoner can do is point them into direction (which is also non-trivial task) and command to destroy everything, yeah